tests/test_inventory_client.py

```diff
@@ -127,6 +187,27 @@ def test_add_team_conflict_error(cli, init_teams):
     assert compare_unsorted_list(cli.teams(), init_teams, lambda x: x.vid)
 
 
+def test_add_team_conflict_error_universe(
+  cli,
+  old_universe,
+  init_old_universe_teams):
```

Comment:
Indent with 4 spaces. Move `):` to a new line.

Reply:
👍